pkgs/tools/misc/tmate/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "tmate-${version}";
-  version = "2.2.0";
+  version = "2.2.1";
 
   src = fetchFromGitHub {
-    owner  = "nviennot";
+    owner  = "tmate-io";
```

Comment:
Well, if you go to https://github.com/nviennot/tmate you'll be redirected to https://github.com/tmate-io/tmate so I'd say the author created an org and moved the repo there. I'm simply reflecting that change.